xgboost-demo-all.ipynb

```diff
@@ -0,0 +1,222 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# XGBoost integration\n",
+    "Here, we present XGBoost integration with Neptune that lets you automatically log metrics (train, eval), save trained model to Neptune and much more.\n",
+    "\n",
+    "Usage is easy: just pass `neptune_callback` to training function like any other xgboost callback.\n",
+    "\n",
+    "You can log multiple data types:\n",
+    "\n",
+    "    * Log metrics (train and eval) after each boosting iteration.\n",
+    "    * Log model (Booster) to Neptune after last boosting iteration.\n",
+    "    * Log feature importance to Neptune as image after last boosting iteration.\n",
+    "    * Log visualized trees to Neptune as images after last boosting iteration.\n",
+    "\n",
+    "## Resources\n",
+    "* Open source implementation is on [GitHub](https://github.com/neptune-ai/neptune-contrib/blob/master/neptunecontrib/monitoring/xgboost_monitor.py),\n",
+    "* Use case / tutorial: https://docs.neptune.ai/integrations/xgboost.html,\n",
+    "* Try integration in action right now. Simply run this Notebook top to bottom. It works outside-the-box :)"
+   ]
+  },
+  {
```

Comment:
We need 

```bash
pip-install ALL THE REQUIREMENTS
```

here.
Remember what where the most common problems people had with our kaggle solution.

Reply:
Yes, I remember :) Adding this